elements/lisk-p2p/src/peer/base.ts

```diff
@@ -128,10 +135,19 @@ export class Peer extends EventEmitter {
 	protected readonly _ipAddress: string;
 	protected readonly _wsPort: number;
 	private readonly _height: number;
-	private _reputation: number;
+	protected _reputation: number;
+	protected _latency: number;
+	protected _connectTime: number;
+	public productivity: {
```

Comment:
Can we also make this one private or protected and expose it as a getter? Also, maybe in the getter we can return a clone of the object to prevent outside code from tempering with the internal productivity object?